pkg/cli/iam_cleanup.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/abcxyz/access-on-demand/apis/v1alpha1"
+	"github.com/abcxyz/access-on-demand/pkg/handler"
+	"github.com/abcxyz/access-on-demand/pkg/requestutil"
+	"github.com/abcxyz/pkg/cli"
+	"github.com/posener/complete/v2/predict"
+
+	resourcemanager "cloud.google.com/go/resourcemanager/apiv3"
+)
+
+var _ cli.Command = (*IAMCleanupCommand)(nil)
+
+// iamCleanupHandler interface that handles the cleanup of IAMRequest.
+type iamCleanupHandler interface {
+	Cleanup(context.Context, *v1alpha1.IAMRequest) ([]*v1alpha1.IAMResponse, error)
+}
+
+// IAMCleanupCommand handles the cleanup of IAM requests, which removed the
+// requested bindings and expires AOD bindings from IAM policy.
+type IAMCleanupCommand struct {
+	cli.BaseCommand
+
+	flagPath string
+
+	flagVerbose bool
+
+	// testHandler is used for testing only.
+	testHandler iamCleanupHandler
+}
+
+func (c *IAMCleanupCommand) Desc() string {
+	return `Cleanup of the IAM request YAML file in the given path, which ` +
+		`removes bindings in the request and expired AOD bindings from the IAM policy`
+}
+
+func (c *IAMCleanupCommand) Help() string {
+	return `
+Usage: {{ COMMAND }} [options]
+
+Cleanup of the IAM request YAML file in the given path:
+
+      {{ COMMAND }} -path "/path/to/file.yaml"
+
+Cleanup of the IAM request YAML file and output applied IAM changes:
+
+      {{ COMMAND }} -path "/path/to/file.yaml" -verbose
+`
+}
+
+func (c *IAMCleanupCommand) Flags() *cli.FlagSet {
+	set := c.NewFlagSet()
+
+	// Command options
+	f := set.NewSection("COMMAND OPTIONS")
+
+	f.StringVar(&cli.StringVar{
+		Name:    "path",
+		Target:  &c.flagPath,
+		Example: "/path/to/file.yaml",
+		Predict: predict.Files("*"),
+		Usage:   `The path of IAM request file, in YAML format.`,
```

Comment:
Unless necessary, we rarely use ` 
```suggestion
		Usage:   "The path of IAM request file, in YAML format.",
```